packages/fxa-react/storybooks.js

```diff
@@ -0,0 +1,42 @@
+const { resolve } = require("path");
+
+const additionalJSImports = {
+  "@fxa-react": __dirname,
+  "@fxa-shared": resolve(__dirname, "../fxa-shared"),
+};
+
+const customizeWebpackConfig = ({ config, mode }) => {
+  // https://storybook.js.org/docs/configurations/typescript-config/#setting-up-typescript-to-work-with-storybook-1
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    loader: require.resolve("babel-loader"),
+    options: {
+      presets: [["react-app", { flow: false, typescript: true }]],
+    },
+  });
+  config.resolve.extensions.push(".ts", ".tsx");
```

Comment:
I suspect, with a rebase, this file should see a few prettier changes (notably quotes)